testsupport/tiers/checks.go

```diff
@@ -43,6 +44,9 @@ var (
 
 func NewChecks(tier string) (TierChecks, error) {
 	switch tier {
+	case base:
```

Comment:
Add basedeactivationdisabled too

Reply:
oops, done in https://github.com/codeready-toolchain/toolchain-e2e/pull/268/commits/6f1b6083896209ac78d010309b31dce485548146